entries/a/also.com.json

```diff
@@ -0,0 +1,17 @@
+{
+  "ALSO Schweiz AG": {
+    "domain": "also.com",
+    "additional-domains": [
+      "also.ch"
+    ],
+    "img": "also.com.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.also.com/ec/cms5/de_6110/6110/services/it-services/multifaktor-authentifizierung/index.jsp",
```

Comment:
If 2FA is mandatory I don't think a documentation link is strictly necessary, and since there doesn't seem to be an English version available it's probably better to remove this line.
```suggestion
```